packages/aurora-editor/test/CardAtBottomEditor.test.js

```diff
@@ -0,0 +1,15 @@
+import { CardAtBottomEditor } from "../index.js";
+import React from "react";
+
+describe("CardAtBottomEditor", () => {
+  it("is exported and exists", () => {
+    expect(CardAtBottomEditor).toBeDefined();
+  });
+
+  it("is a valid React element", () => {
+    const element = (
+      <CardAtBottomEditor onSubmit={() => {}} onChange={() => {}} />
+    );
+    expect(React.isValidElement(element)).toBe(true);
```

Comment:
What does this test do exactly?

Reply:
I was having bugs where some things weren't being exported and rendered correctly as subclasses of React Components, so I put this in here so it wouldn't happen again. 

Basically it just checks if the item is rendering without any errors.